app/views/users/index.html.erb

```diff
@@ -1,2 +1 @@
-<h1>Here your users list</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+<%= render "user" %>
```

Comment:
- [ ] kindly match the design given, currently images are bigger than image div and also the screen
![Screenshot from 2023-05-20 07-15-43](https://github.com/Becky449/Myblog/assets/66097371/687eb98a-9d1b-4e5d-b462-19763b0a092a)